src/cloudShellRunner.ts

```diff
@@ -135,6 +135,7 @@ export class CloudShellRunner extends BaseRunner {
 
                 this._outputChannel.appendLine('Connecting to Cloud Shell failed with error: \n' + err);
                 this._outputChannel.show();
+                return;
```

Comment:
why this return needed? seems this function exist smoothly

Reply:
if provisioned failed, no  need to run next file uploading step.